spec/factories/targets.rb

```diff
@@ -6,5 +6,6 @@
     target_count 100
     unit Target.units[:submissions]
     conference
+    campaign
   end
```

Comment:
Thank you. I have used `campaign_target` as name of new factory. Let me know if you can think of better name.